hack/gofmt.sh

```diff
@@ -10,7 +10,7 @@ if [ "${IS_CONTAINER}" != "false" ]; then
   export XDG_CACHE_HOME="/tmp/.cache"
 
   cd "${TOP_DIR}"
-  DIRS=$(make show_dirs)
+  DIRS="$(go list -f='{{ .Dir }}' ./...)"
```

Comment:
`-l` is not a recognized option for `go fmt`, only for `gofmt`. 🤷

Reply:
TIL those both exist.

Anyhow, my point was that `gofmt -l .` works fine and we don't need to write any code that lists all of the modules just to feed them back in again.

The only reason we were doing the `make show_dirs` thing was because we wanted to avoid running gofmt on the operator-sdk.

Reply:
Fixed.